src/screen/react-nativeconfig.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    assets: ['./assets/Fonts']
+}
```

Comment:
Tooo many issues in this - 
1. This should be in root folder.
2. path is wrong for fonts
3. file name is wrong, it should be `react-native.config.js`